README.md

```diff
@@ -1,21 +1,21 @@
-# Shotgun Enterprise Toolbox This repository contains various tools that can be useful for Shotgun System Administrators
+# ShotGrid Enterprise Toolbox This repository contains various tools that can be useful for ShotGrid System Administrators
```

Comment:
I think this is actually two lines somehow merged into one. 

It's most likely this:
```
# ShotGrid Enterprise Toolbox

This repository contains various tools that can be useful for ShotGrid System Administrators
```

Reply:
Good catch. Added a line break.